scripts/integration/run-binchain.sh

```
@@ -68,8 +74,8 @@ for scenario in ${SCENARIOS}; do
   echo "═══ ${scenario}: fresh chain ═══"
   start_chain
   if ! nix develop "${HOPRNET_SHELL:-github:hoprnet/hoprnet}" -c \
-    cargo test --manifest-path integration/Cargo.toml --test integration "${scenario}" \
-    --no-fail-fast -- --include-ignored --test-threads=1; then
+    cargo test --manifest-path integration/Cargo.toml --test "${TEST_TARGET}" "${scenario}" \
+    --no-fail-fast -- --include-ignored --test-threads=1 ${TEST_ARGS}; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)